src/io/flutter/logging/FlutterLogEntry.java

```diff
@@ -6,16 +6,19 @@
 package io.flutter.logging;
 
 import com.intellij.openapi.util.text.StringUtil;
+import io.flutter.logging.util.LineInfo;
+import io.flutter.logging.util.StyledText;
 import org.jetbrains.annotations.NotNull;
 import org.jetbrains.annotations.Nullable;
 
-import static io.flutter.logging.FlutterLogEntryParser.TOOLS_CATEGORY;
+import java.util.List;
 
 public class FlutterLogEntry {
 
   public enum Kind {
     RELOAD,
     RESTART,
+    WIDGET_ERROR_START,
```

Comment:
nit: a little bit of a surprising log entry kind but I don't see a much better option.

Reply:
Yeah.  Not at all ideal.  But maybe so glaringly odd that it keeps the pressure on to fix this more properly at the framework level...